### PR TITLE
feature/411

### LINKED ISSUE #411: Remove departure window blue line
Deprecate `CanvasController.canvas_draw_aircraft_departure_window()`, because it isn't used anymore.

image

### PR DESCRIPTION
**Pull request by sam-irl**
Removes "departure window" and the blue line at the edge of the radar screen, resolves #411

### CODE REVIEW

File: src/assets/scripts/client/canvas/CanvasController.js [hunk 1]
```diff
@@ -758,29 +758,6 @@ export default class CanvasController {
...
     }
 
-    // TODO: This likely has no purpose, and should be removed.
-    /**
-     * @for CanvasController
-     * @method canvas_draw_aircraft_departure_window
-     * @param cc
-     * @param aircraft
-     */
-    canvas_draw_aircraft_departure_window(cc, aircraft) {
```
Comment (n8rzz) [inline src/assets/scripts/client/canvas/CanvasController.js]: Does this method get called anywhere? Or was this really an unused method?

Comment (sam-irl) [inline src/assets/scripts/client/canvas/CanvasController.js]: No; it isn't called anywhere.

**Comment by sam-irl** [comment]
@openscope/simulator-admins this can be merged

File: src/assets/scripts/client/canvas/CanvasController.js (hunk 1 above)
Comment (erikquinn) [inline src/assets/scripts/client/canvas/CanvasController.js]: `CanvasController.js:785`

Comment (sam-irl) [inline src/assets/scripts/client/canvas/CanvasController.js]: missed that, thanks

**Pull request merged**